tests/docker_run_tests.rs

```diff
@@ -253,6 +253,9 @@ fn test_yarn_custom_version() {
 fn test_yarn_berry() {
     let name = simple_build("./examples/node-yarn-berry");
     let output = run_image(name, None);
+
+    println!("Output: {}", output);
```

Comment:
Why we need to log output?

Reply:
We do not, I was just testing stuff. Good catch